server/auth/login.go

```diff
@@ -8,13 +8,12 @@ import (
 	"github.com/8tomat8/SSU-Golang-252-Chat/loger"
 	"github.com/8tomat8/SSU-Golang-252-Chat/messageService"
 	"golang.org/x/crypto/bcrypt"
-	"github.com/gorilla/websocket"
 )
 
-//+token
 func getUserByName(UserName string) (*messageService.User, error) {
 
 	db, err := database.GetStorage()
+	defer db.Close()
```

Comment:
It is not good idea... On second call of this function you will get closed connection.